jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -528,9 +528,17 @@ def _calc_background_median(self, reg, data=None):
                 data = self.dataset.selected_dc_item
 
         comp = data.get_component(data.main_components[0])
+        if self.config == "cubeviz" and data.ndim > 2:
+            if "spectral_axis_index" in getattr(data, "meta", {}):
+                spectral_axis_index = data.meta["spectral_axis_index"]
+            else:
+                spectral_axis_index = 0

```

Comment:
Too convoluted as a single line? 🤷‍♂️ 

```suggestion
            spectral_axis_index = getattr(data, "meta", {}).get("spectral_axis_index", 0)
```